Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1016,6 +1027,53 @@ def genS(fragment,howMuch):
     key = "FS_" + x + "_UP15_PU25"
     steps[key] = merge([{"cfg":steps[x]["cfg"]},FS_UP15_PU25_OVERLAY])
 
+### FastSim: produce sample of premixed minbias events UP17
+steps["FS_PREMIXUP17_PU50"] = merge([
+        {"cfg":"SingleNuE10_cfi",
+         "--fast":"",
+         "--conditions":"auto:phase1_2017_realistic",
+         "-s":"GEN,SIM,RECOBEFMIX,DIGI",
```

Comment:
```suggestion
         "-s":"GEN,SIM,RECOBEFMIX,DIGI,L1,DIGI2RAW",
```
because #24184 is not in 10_2_X (see also #25513).